src/components/newUIComponents/nav/Navbar2.js

```diff
@@ -0,0 +1,53 @@
+import { NavLink } from "react-router-dom";
+import { LuSun } from "react-icons/lu";
+import logo from "../../../assets/logo.svg";
+import React from "react";
+
+export default function Navbar2({ props }) {
+  return (
+    <div className="Navbar2">
+      <div className="brand">
+        <img src={logo} className="brand-img" alt="brand" />
+      </div>
+      <nav className="links">
+        <NavLink
+          to="/"
+          className={({ isActive }) => (isActive ? "link-active" : "link")}
```

Comment:
Why not use the provided `active` class on the `NavLink` instead of defining our own each time?